App.js

```diff
@@ -1,76 +1,26 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
 import HomePage from "./Pages/HomePage";
 import TrackFood from "./Pages/TrackFood";
 import Progress from "./Pages/Progress";
 import WeightLog from "./Pages/WeightLog";
-import { NavigationContainer, useNavigation } from "@react-navigation/native";
+import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
 import TestPage from "./Pages/Testpage";
-import { useState, useEffect, createContext } from "react";
-import * as SQLite from "expo-sqlite";
-import { DBContext } from "./DbContext";
+import DataBaseContext from "./DbProvider";
```

Comment:
if `DataBaseContext` is actually a provider then it should just be named `DbProvider`

Reply:
wait though this should already be in master branch from the last PR merge, I'm guessing the rebase didn't work correctly